src/routes/api.js

```diff
@@ -11,4 +12,10 @@ router.get('/status/:result_id', async (req, res) => {
   return response
 })
 
+router.get('/get-lpa-boundary/:boundaryId', async (req, res) => {
```

Comment:
Can you add JSDoc that specified what shape of returned data (including error scenarios)?